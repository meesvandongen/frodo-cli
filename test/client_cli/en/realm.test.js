import cp from 'child_process';
import { promisify } from 'util';

const exec = promisify(cp.exec);
const CMD = 'frodo realm --help';
const { stdout } = await exec(CMD);

test("CLI help interface for 'realm' should be expected english", async () => {
  expect(stdout).toMatchSnapshot();
});
