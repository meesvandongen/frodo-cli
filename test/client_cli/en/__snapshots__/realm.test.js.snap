// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI help interface for 'realm' should be expected english 1`] = `
"Usage: frodo realm [options] [command]

Manage realms.

Options:
  -h, --help            Help

Commands:
  add-custom-domain     Add custom domain (realm DNS alias).
  describe|details      Describe realms.
  help [command]        display help for command
  list                  List realms.
  remove-custom-domain  Remove custom domain (realm DNS alias).
"
`;
